notification/tests.py

```diff
@@ -15,7 +15,29 @@ def test_delete_notification_from_database(self, notification):
         with pytest.raises(Notification.DoesNotExist):
             Notification.objects.get(pk=notification.pk)
 
-    def test_mark_as_read_notification(self, notification):
+    def test_update_notification_message(self, notification):
+        new_notification = notification
```

Comment:
Is this variable necessary here? If you only use the notification fixture and updated_notification it will not works?

Reply:
There is the line:
`updated_notification = Notification.objects.get(pk=new_notification.pk)`.
If I'll remove new_notification, and instead I'll write `updated_notification = Notification.objects.get(pk=notification.pk)`, the test `assert new_notification.id == notification.id == updated_notification.id` would blindly pass.
What about a case, that for some reason, the notification.id has been changed?
Without `new_notification` variable we'll miss that the ID has been changed. And as for now, we don't want that the notification Id would change if we just changed the message of it.

Reply:
I was wrong. This is not a copy, but it's a new pointer. I updated the code to use a copy of the ID instead of using another pointer. Please have a look :-)